.travis.yml

```diff
@@ -16,18 +16,20 @@ matrix:
       gemfile: Gemfile
     - rvm: 2.1.0
       gemfile: Gemfile
-    - rvm: 2.1.1
-      gemfile: Gemfile
     - rvm: 2.2.0
       gemfile: Gemfile
-    - rvm: 2.3.1
+    - rvm: 2.3.0
       gemfile: Gemfile
-    - rvm: 2.4.5
+    - rvm: 2.4.0
       gemfile: Gemfile
-    - rvm: 2.5.3
+    - rvm: 2.5.0
       gemfile: Gemfile
     - rvm: 2.6.0
       gemfile: Gemfile
+    - rvm: 2.7.0
+      gemfile: Gemfile
+    - rvm: 3.0.0
+      gemfile: Gemfile
 
 notifications:
   email: false
```

Comment:
Also the IRC channel on Freenode looks dead, should I remove this notification?

Reply:
given most of the projects are moving away from freenode and the channel has noone interested it would make sense to me to drop.

Reply:
It was actually removed from the project site, no IRC at the moment. I will remove this.